config-schema.json

```diff
@@ -455,6 +455,21 @@
                 ],
                 "pattern": ""
               },
+              "custom_features": {
+                "$id": "#/properties/entities/properties/remote/items/properties/custom_features",
+                "type": "array",
+                "title": "The custom_features Schema",
+                "items": {
+                  "$id": "#/properties/entities/properties/remote/items/properties/custom_features/items",
+                  "type": "string",
+                  "title": "The Custom Features Schema",
+                  "default": "",
+                  "examples": [
+                    "Button1, Button2, Button3"
+                  ],
+                  "pattern": ""
```

Comment:
Either define a pattern or remove it. An empty pattern adds no value.
(In the next cleanup, I'll remove the other empty patterns in the schema).

Reply:
I will remove that.